.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{kt,kts}]
+indent_size=4
+continuation_indent_size=4
```

Comment:
I'm not sure if `continuation_indent_size` is still supported. Check this issue: https://github.com/pinterest/ktlint/issues/308

Reply:
It seems you are correct and the option is not used anymore. Removed that in 186d17d.